HoloPipelines/core/tasks/abdominal_organs_segmentation/split_to_separate_organs.py

```diff
@@ -1,14 +1,15 @@
 """
 This module contains the functionality of the abdominal_organs_segmentation pipeline
 of splitting the combined result into multiple sub-results.
-"""
 
-# TODO: Depending on how we decide for this pipeline, this component may get removed.
-# Otherwise, separate organs need to be merged again! (can also be in this task, which
-# should then be renamed though)
-# TODO: If we keep this, refactor and verify
-# Refactoring should include only doing the OBJs in here, and transforming to GLB on
-# pipeline level
+Currently this module is not in used. The current implementation of
+this module is unable to provide an appropriate name to each output
```

Comment:
Hm actually we can provide a name that's not the problem. Problem is we can't assign colours, and can't merge them. But this can be made shorter:

"Currently, this component is unused, as the abdominal_organs_segmentation pipeline yields only one output. This might be integrated later. Note that a further step will need to be implemented, to either merge the 8 models back into 1, or make the pipeline yield 8 outputs for one job."